src/libfuncs/box.rs

```diff
@@ -73,18 +69,7 @@ pub fn build_into_box<'ctx, 'this>(
     let inner_type = registry.get_type(&info.ty)?;
     let inner_layout = inner_type.layout(registry)?;
 
-    let value_len = entry
-        .append_operation(arith::constant(
-            context,
-            IntegerAttribute::new(
-                IntegerType::new(context, 64).into(),
-                inner_layout.pad_to_align().size().try_into()?,
-            )
-            .into(),
-            location,
-        ))
-        .result(0)?
-        .into();
+    let value_len = entry.const_int(context, location, inner_layout.size(), 64)?;
```

Comment:
I think you forgot the `.pad_to_align()`.

Reply:
@azteca1998 fixed.